guides/common/modules/ref_customizing-job-templates.adoc

```diff
@@ -1,10 +1,10 @@
-[id="customizing-templates_{context}"]
+[id="Customizing_Job_Templates_{context}"]
 = Customizing Job Templates
 
 When creating a job template, you can include an existing template in the template editor field.
 This way you can combine templates, or create more specific templates from the general ones.
 
-The following template combines default templates to install and start the *httpd* service on {RHEL} systems:
+The following template combines default templates to install and start the *httpd* service on clients:
```

Comment:
Can we come up with a different example service than `httpd` which famously has different package names across Linux distributions?

Reply:
OK, the obvious solution is `nginx`. Any objections?

Reply:
No objections from me.